File: src/main/java/com/realyoungk/sdi/SdiApplication.java
package com.realyoungk.sdi;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@ConfigurationPropertiesScan
@EnableScheduling
public class SdiApplication {

	public static void main(String[] args) {
		SpringApplication.run(SdiApplication.class, args);
	}

}

File: src/main/kotlin/com/realyoungk/sdi/SdiApplication.kt
package com.realyoungk.sdi

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@ConfigurationPropertiesScan
@EnableScheduling
class SdiApplication

fun main(args: Array<String>) {
    runApplication<SdiApplication>(*args)
}